Package.swift

```diff
@@ -45,6 +45,20 @@ func generateDependencies() -> [Package.Dependency] {
     }
 }
 
+// To develop this on Apple platforms, set this to true
+let development = false
+let swiftSettings: [SwiftSetting]
+if development {
+    swiftSettings = [
+        .define("CRYPTO_IN_SWIFTPM"),
+        .define("CRYPTO_IN_SWIFTPM_FORCE_BUILD_API"),
+    ]
+} else {
+    swiftSettings = [
+        .define("CRYPTO_IN_SWIFTPM"),
+    ]
+}
```

Comment:
None of this should be necessary: the Package.swift in swift-crypto exists to support the same code being part of CryptoKit and swift-crypto, which is not a problem that exists here.